kogitoq/gadaq/src/main/java/life/genny/gadaq/live/data/InternalConsumer.java

```diff
@@ -96,6 +100,10 @@ public void getData(String data) {
 		msg.setToken(userToken.getToken());
 		KafkaUtils.writeMsg(KafkaTopic.GENNY_DATA, msg);
 
+		if(filter.isValidEvent(msg)){
```

Comment:
Please Move this to `Events.java`

Reply:
Add event route supporting QDataAnswerMessage